build.sbt

```diff
@@ -1,16 +1,19 @@
 name := "spark-json-schema"
 
-version in ThisBuild := "0.2"
+version in ThisBuild := "0.3"
 organization := "org.zalando"
 
 scalaVersion := "2.11.8"
 
-libraryDependencies += "org.apache.spark" %% "spark-sql" % "1.6.2"  % Provided
-libraryDependencies += "com.typesafe.play" %% "play-json" % "2.5.4"
-dependencyOverrides ++= Set("com.fasterxml.jackson.core" % "jackson-databind" % "2.4.4")
+libraryDependencies += "org.apache.spark" %% "spark-sql" % "2.0.2"  % Provided
```

Comment:
we can only use `2.0.1` for now, since the spark appliance is not yet ready for `2.0.2`